brave/browser-laptop
# Refactor Dialog with Aphrodite

Closes #8984

Per https://github.com/brave/browser-laptop/commit/2dd92a9eb3b42053ceaafc32a63e1d3508e8c039, dialog.less can be safely removed (as styles under 'menu' has been just copied).

Test Plan:
1. Open 'Import Browser Data...' from Bookmarks on the main menu
2. Open devtool
3. Make sure 'dialog' is styled properly

Submitter Checklist:


Reviewer Checklist:

Tests
